### DISCUSSION TITLE
Specifying an owner of data

### DISCUSSION THREAD

**Discussion by FallenRiteMonk** [serverpod/serverpod#788] [Q&A] [answered]
Hy,

i would like to add the owner (creater) of database entries. If i add the colomn: `ownerId: int, parent=serverpod_user_info` to my tables yaml file i get the error that the table does not exist. Can anybody tell me what im doing wrong?

**Comment by fischerscode** [comment, marked as answer]
Unfortunately, this is a known limitation (#695). So for now, your only option would be adding the foreign key manually.
